install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -207,6 +207,17 @@ containers:
     value: |
            {{ toJSON .ObjectMeta.Labels }}
   {{ end }}
+  {{- if .DeploymentMeta.Name }}
+  - name: ISTIO_META_WORKLOAD_NAME
+    value: {{ .DeploymentMeta.Name }}
+  {{ end }}
+  {{- if and .TypeMeta.APIVersion .DeploymentMeta.Name }}
+  - name: ISTIO_META_OWNER
+    value: kubernetes://api/{{ .TypeMeta.APIVersion }}/namespaces/{{ valueOrDefault .DeploymentMeta.Namespace `default` }}/{{ toLower .TypeMeta.Kind}}/{{ .DeploymentMeta.Name }}
+  {{- end}}
+  - name: ISTIO_METAJSON_PORTS_TO_CONTAINERS
```

Comment:
Note that Louis wants to get rid of the container ports ( even for 1.3 ). Ok to pass if they're there, but nothing should assume they are.

Reply:
Yes. This is meant to ensure that existing functionality is passed forward. We will need to adjust assumptions about removing container port requirements as that happens. That action *will* have an impact on metrics collection.